The job **`x86_64-gnu-tools`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/11532314052/job/32103987361) [(plain)](https://github.com/rust-lang/rust/commit/ae66380da0d219da669e94b7e591d6853191dd0e/checks/32103987361/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
.......... (130/133)
..F        (133/133)


/checkout/tests/rustdoc-gui/docblock-big-code-mobile.goml docblock-big-code-mobile... FAILED
[ERROR] `tests/rustdoc-gui/docblock-big-code-mobile.goml` line 9: The following errors happened (for selector `.docblock p > code`): [expected `44` for property `"offsetHeight"`, found `48`]: for command `assert-property: (".docblock p > code", {"offsetHeight": "44"})`
[ERROR] `tests/rustdoc-gui/docblock-big-code-mobile.goml` line 13: The following errors happened (for selector `.docblock p > a > code`): [expected `44` for property `"offsetHeight"`, found `48`]: for command `assert-property: (".docblock p > a > code", {"offsetHeight": "44"})`
Error: ()
Build completed unsuccessfully in 0:03:35
  local time: Sat Oct 26 14:19:42 UTC 2024
  network time: Sat, 26 Oct 2024 14:19:42 GMT
```

</details>

--- a/tests/rustdoc-gui/docblock-big-code-mobile.goml
+++ b/tests/rustdoc-gui/docblock-big-code-mobile.goml
@@ -1,13 +1,15 @@
 // If we have a long `<code>`, we need to ensure that it'll be fully displayed on mobile, meaning
 // that it'll be on two lines.
+
 emulate: "iPhone 8" // it has the following size: (375, 667)
 go-to: "file://" + |DOC_PATH| + "/test_docs/long_code_block/index.html"
-// We now check that the block is on two lines:
 show-text: true // We need to enable text draw to be able to have the "real" size
+
+// We now check that the block is on two lines:
 // Little explanations for this test: if the text wasn't displayed on two lines, it would take
-// around 20px (which is the font size).
-assert-property: (".docblock p > code", {"offsetHeight": "44"})
+// around 24px (which is the font size).
+assert-size: (".docblock p > code", {"height": 48})
 
 // Same check, but where the long code block is also a link
 go-to: "file://" + |DOC_PATH| + "/test_docs/long_code_block_link/index.html"
-assert-property: (".docblock p > a > code", {"offsetHeight": "44"})
+assert-size: (".docblock p > a > code", {"height": 48})

